src/Aspire.Cli/api/aspire.cs

```diff
@@ -0,0 +1,15 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Aspire.Cli
```

Comment:
@joperezr can we skip this dependency.

Reply:
Opened https://github.com/dotnet/aspire/pull/8221 to fix this.

Reply:
@joperezr - is there a reason this still shows up? Is it because it was already there? Or is your fix not working?

Reply:
Seems like it isn't working, will take a closer look.

Reply:
Fixed.